Provide a simpler html start file for trying your own data

davidnmora opened the pull request:
There's a lot to take in with DeepScatter, especially in the `index.html` file. Web dev is my day job, but it still took me a good while to weed through it all & figure out what I needed for my own data. 

This file provides a good place to just see the basics.

One aspect to address: making it clear how to point to it when you run `npm run dev` as opposed to `index.html`

bmschmidt reviewed (approved):
Thanks, this is a really useful addition! We should probably also list it in the quickstart.

davidnmora commented:
@bmschmidt Sounds good!

I updated the README, & also added a `npm run dev-simple` command.

I'm not particularly attached to the extra command, so happy to nix it, just thought it might make set up that much easier.

davidnmora commented on vite.config.cjs:
```diff
@@ -1,30 +1,35 @@
+    open:
+      mode === "simple" ? "/index-simplest-way-to-start.html" : "/index.html",
+  },
   build: {
```
^ this is the change.

I just had to use defineConfig in order to read off mode.

davidnmora commented:
@bmschmidt good point! I've updated the README with instructions.

Note that I've dropped your notes about changing the fields in `encoding` from the section I edited. I think it'll be clear when contrasting the simplified example with `index.html` what data-specific fields are being introduced, so it should be fine.

bmschmidt reviewed (approved):
Thanks so much!
